api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -137,7 +137,13 @@ func (in *TenantSpec) DeepCopyInto(out *TenantSpec) {
 		*out = make([]string, len(*in))
 		copy(*out, *in)
 	}
-	in.TenantResource.DeepCopyInto(&out.TenantResource)
+	if in.TenantResources != nil {
```

Comment:
is this file still auto generated by using the k8s client something? you could add this file to `.gitattributes` so it hides automatically for reviewers.

something like:
```
api/v1alpha1/zz_generated.deepcopy.go linguist-generated
```

Reply:
good idea -- will include that in next PR